docs/qa/v034/README.md

```diff
@@ -0,0 +1,251 @@
+The plot below shows the evolution over time of the cumulative number of transactions inside all full nodes' mempools.
+The two spikes that can be observed correspond to a period where consensus instances reached round 1
+at some nodes.
+
+![mempool-cumulative](./img/v034_r400c8_mempool_size.png)
+
+The plot below shows evolution of the average over all full nodes, which oscillate around 140 outstanding transactions.
+
+![mempool-avg](./img/v034_r400c8_mempool_size_avg.png)
+
+The peaks observed coincide with the moments when some nodes process to round 1 of consensus (see below).
+
+### Peers
+
+The number of peers was stable at all nodes.
+It was higher for the seed nodes (around 140) than for the rest (between 21 and 71).
+
+![peers](./img/v034_r400c8_peers.png)
+
+### Consensus Rounds per Height
+
+Most heights took just one round, but some nodes needed to advance to round 1 at some point.
+
+![rounds](./img/v034_r400c8_rounds.png)
+
+### Blocks Produced per Minute, Transactions Processed per Minute
+
+The blocks produced per minute are the gradient of this plot.
+
+![heights](./img/v034_r400c8_heights.png)
+
+Over a period of 2 minutes, the height goes from 967 to 1033.
+This results in an average of 33 blocks produced per minute.
+
+The transactions processed per minute are the gradient of this plot.
+
+![total-txs](./img/v034_r400c8_total-txs.png)
+
+Over a period of 2 minutes, the total goes from 25362 to 31472 transactions,
+resulting in 3055 transactions per minute. However, we can see in the plot that
+all transactions in the load are process long before the two minutes.
+If we adjust the time window when transactions are processed (approx. 92 seconds),
+we obtain 3984 transactions per minute.
+
+### Memory Resident Set Size
+
+Resident Set Size of all monitored processes is plotted below.
+
+![rss](./img/v034_r400c8_rss.png)
+
+The average over all processes oscillates around 420 MiB and does not demonstrate unconstrained growth.
+
+![rss-avg](./img/v034_r400c8_rss_avg.png)
+
+### CPU utilization
+
+The best metric from Prometheus to gauge CPU utilization in a Unix machine is `load1`,
+as it usually appears in the
+[output of `top`](https://www.digitalocean.com/community/tutorials/load-average-in-linux).
+
+![load1](./img/v034_r400c8_load1.png)
+
+It is contained between 0.3 and 4.2 at all nodes.
+
+## Test Result
+
+**Result: N/A**
+
+Date: 2022-09-23
+
+Version: a41c5eec1109121376de3d32379613856d4a47dd
+
+# Rotating Node Testnet
+
+We reuse the work done on the 200 node testnet where we found the saturation point,
+and will thus focus on the load `c=4,r=800`.
+
+# Latencies
+
+The plot of all latencies can be seen in the following plot.
+
+![rotating-all-latencies](./img/v034_rotating_latencies.png)
+
+We can observe there are some very high latencies, towards the end of the test.
+Upon suspicion that they are duplicate transactions, we examined the latencies
+raw file and discovered there are more than 100K duplicate transactions.
```

Comment:
Why were there so many duplicate transactions? I thought the scheme we were using with the load generation tool would ensure uniqueness among the transactions by way of the _value_ component of the key/value pairs?

Reply:
The reason why I ask this, is that this could explain why we had so few valid transactions during the tests. This skews the throughput rate calculation.

Reply:
The load generator produces unique transactions, the proof is that `v0.37` doesn't contain any duplicate TX. The impression I have is that somehow some transactions are gossiped around and keep on being accepted in blocks. I know there is a protection built into the memepool to prevent this, but I guess it is not working all the time.
Gut feeling is that there is a bug that was triggered in the `v0.34` run, but not in the `0.37` run. My next step on this is to look into @jmalicevic's test to see if the reason for what she saw is also duplicate TXs. If that is the case, I will add some info to the issue she wrote, else, I'll create another issue.

Reply:
Could it be that the same issue that affects the 200 node network at high load also affected this test? I.e. that the mempool cache is not large enough and so duplicate transactions end up being submitted.

Reply:
I think so.